graphs/minimum_spanning_tree_prims.py

```diff
@@ -3,6 +3,16 @@
 
 
 def prisms_algorithm(l):  # noqa: E741
+    """
+    >>> l = {0: [[1, 1], [3, 3]], \
+             1: [[0, 1], [2, 6], [3, 5], [4, 1]], \
+             2: [[1, 6], [4, 5], [5, 2]], \
+             3: [[0, 3], [1, 5], [4, 1]], \
+             4: [[1, 1], [2, 5], [3, 1], [5, 4]], \
+             5: [[2, 2], [4, 4]]}
```

Comment:
PEP8: Avoid backslash line termination in Python because a single whitespace to the right of the backslash breaks the script on a change that is invisible to the reader.

Backslashes are not needed between ( ), [ ], { }.

Reply:
While here, please fix the [flake8 E741](https://pycodestyle.pycqa.org/en/latest/intro.html#error-codes) issues.